src/sass/components/_footer.scss

```diff
@@ -67,64 +63,78 @@
   position: relative;
   transition: color 500ms ease-in-out, transform 500ms ease-in-out;
   transition: font-weight 500ms ease-in-out, transform 500ms ease-in-out;
```

Comment:
Uważam, jednak że `font-weight` tutaj bardzo psuje animację.
Prosiłbym cię o usunięcie tej animacji
`font-weight 500ms ease-in-out`